src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/modalwindows/FacebookSignupModalComponentObject.java

```diff
@@ -71,26 +69,24 @@ public Boolean apply(WebDriver input) {
     Object[] handles = driver.getWindowHandles().toArray();
 
     driver.switchTo().window(handles[1].toString());
-    wait.forElementVisible(editInfoProvided);
-    editInfoProvided.click();
-    PageObjectLogging.log("acceptWikiaAppPolicyNoEmail", "editing info provided", true);
-    wait.forElementVisible(emailCheckbox);
-    emailCheckbox.click();
-    PageObjectLogging.log("acceptWikiaAppPolicyNoEmail", "unchecked the email checkboxbox", true);
+//    wait.forElementVisiblentVisible(editInfoProvided);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3ACommentedOutCodeLine)